test/unit/player.test.js

```diff
@@ -283,6 +281,7 @@ QUnit.test('should asynchronously fire error events during source selection', fu
   });
 
   this.clock.tick(1);
+  this.clock.tick(1);
```

Comment:
It's because the error is triggered async, so, we need one for it, but we need an additional one for the player creation itself.